assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -965,7 +966,8 @@ export default Component.extend({
     }
 
     this.element.querySelectorAll(".lazyYT").forEach((iframe) => {
-      $(iframe).lazyYT();
+      $(iframe).lazyYT({ collapsible: true });
```

Comment:
oh wait this shouldn't be here

Reply:
I'm also thinking the caret when it's open should point up, to show that you can collapse it, and then when collapsed, the caret should point down